templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,20 @@
 # Press Enter to send
 
-By default, Zulip sends your message when you press enter (or return)
-on your keyboard. If you want to turn this off, simply uncheck the
-**Press Enter to send** box below the compose field:
+By default, Zulip sends your message when you press the **Send** button in
+the compose box. If you would like to send your message by pressing
+the enter (or return) key, follow these steps:
 
-![Box checked](/static/images/help/enter-box-checked.png)
+1. Check the **Press Enter to send** box below the compose field:
 
-You will now see a **Send** button below the compose field looking
-like this.
+  ![Box unchecked](/static/images/help/enter-box-unchecked.png)
```

Comment:
The image should be indented by 4 spaces; that's why it's not rendering correctly.